consultation_analyser/pipeline/backends/langchain_llm_backend.py

```diff
@@ -30,8 +30,8 @@ def summarise_theme(self, theme: models.Theme) -> ThemeSummary:
         )
 
         prompt_inputs = {
-            "consultation_name": theme.question.section.consultation.name,
-            "question": theme.question.text,
+            "consultation_name": theme.topic_model.question.section.consultation.name,
+            "question": theme.topic_model.question.text,
```

Comment:
Think this is a vestige of when we went theme > model > question? it causes a crash when I run locally

Reply:
Yes, updated (and I've run the pipeline locally against Sagemaker and running ok).

Reply:
I'll raise a PR to add a test that exercises the moving parts of the langchain backend with a `FakeListLLM`